index.html

```diff
@@ -1,237 +1,258 @@
+          <li class="language">RSpec</li>
+
+          <li class="language">CapyBara</li>
+          <li class="language">Selenium</li>
+          <li class="language">JBootstrap</li>
+          <li class="language">Ruby on Rails</li>
+        </div>
+      </ul>
+      <ul class="skill_list flex">
+        <li class="Language_skill3"><img class="border_line" src="image folder/Rectangle57.png" alt="rectangle_icon">
+          Skills
+        </li>
+        <div class="Languge_skill_list3">
+          <li class="language">Codekit</li>
+          <li class="language">Github</li>
+
+          <li class="language">Gitlab</li>
+          <li class="language">Terminal</li>
+          <li class="language">Codepen</li>
+        </div>
+      </ul>
+
+
+    </div>
+  </section>
+  <section id="form_container_link Contact" class="form_section">
+    <div class="form_container">
+      <h3 class="form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+        touch..</h2>
+
         <form class="contact_me_form" action="https://formspree.io/f/xoqbwpby" method="post">
 
-             <input class="form_input_fnames" name= "first_name" type="text" placeholder="First name" maxlength="30" required>           
-            <input class="form_input_lnames" name= "last_name" type="text" placeholder="Last name" maxlength="30" required>
-            <input class="form_input_names" name= "Full names" type="text" placeholder="Full names" maxlength="30" required>
-            <hr class="fullname_line">
-            <br>
-            <br>                          
-            <input class="form_input_email" name= "email_address" type="email" placeholder="abcd@form.com" required>
-            <hr class="fullname_line">
-            <br>
-            <textarea class="form_input_textarea" name="comment_message" id="" cols="30" rows="5" placeholder="Leave us a message" maxlength="500" required></textarea> <br>
-            <button type="submit" name= "submit"class="get_in_touch_button">Get in touch</button>
-           
+          <input class="form_input_fnames" name="first_name" type="text" placeholder="First name" maxlength="30"
+            required>
+          <input class="form_input_lnames" name="last_name" type="text" placeholder="Last name" maxlength="30" required>
+          <input class="form_input_names" name="Full names" type="text" placeholder="Full names" maxlength="30"
+            required>
+          <hr class="fullname_line">
+          <br>
+          <br>
+          <input class="form_input_email" name="email_address" type="email" placeholder="abcd@form.com" required>
+          <hr class="fullname_line">
+          <br>
+          <textarea class="form_input_textarea" name="comment_message" id="" cols="30" rows="5"
+            placeholder="Leave us a message" maxlength="500" required></textarea> <br>
+          <button type="submit" name="submit" class="get_in_touch_button">Get in touch</button>
```

Comment:
- [ ] I suggest that In order to improve further the accessibility of your page you modify the labels of the required fields of your contact form as explained below.

Following the [Easy Checks](https://www.w3.org/WAI/test-evaluate/preliminary/#forms) accessibility instructions for forms:

![image](https://user-images.githubusercontent.com/77069011/207383718-17a0b676-3bf8-43c4-ac44-2f2852e26514.png)

Please add an indicator of which fields are required! 🤓 🛠️